### ISSUE TITLE
Error while searching for regular expression

### ISSUE THREAD

**Issue by darnn** [SubtitleEdit/subtitleedit#9158]
![image](https://github.com/user-attachments/assets/5ebfb40d-b71c-4e00-9e7d-0c273dd76ad6)

Also, is there actually a way to do what I'm trying to do there (i.e. include all Latin characters)?

**Comment by niksedk** [comment]
Try `\p{L}`

`\p{Lu}` will match an uppercase letter that has a lowercase variant
`\p{Ll}` will match a lowercase letter that has an uppercase variant

**Comment by darnn** [comment]
Ah, yes, that's a thought... Is there a way to do it for other languages, though? With English and Hebrew I've just been doing it the long way so far, `([אבגדהוזחטיכלמנספצקרשת])([abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ])`.

**Comment by markfilipak** [comment]
Instead of ([abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ]) use ([a-z]|[A-Z]).

**Comment by darnn** [comment]
Yeah, I thought I had some specific reason I was doing it like that rather than [a-z], but either it's long been resolved or I was just doing it wrong back then.

**Comment by darnn** [comment]
Actually, to my shame, I asked ChatGPT, and it was actually useful. Apparently you can use these for specific languages:
https://www.lookuptables.com/text/unicode-characters
In the form of `[\u0590-\u05FF]`, `[\u0400-\u04FF]` and so on.
